frontend/client/types.ts

```diff
@@ -33,8 +33,10 @@ export type ChainData = {
     agent_id: number;
     cost_of_bond: number;
     fund_requirements: {
-      agent: number;
-      safe: number;
+      [tokenSymbol: string]: {
```

Comment:
it should be token address actually, or is it just for labelling?

Reply:
oh right good catch